### ISSUE TITLE
Add alt text for images on the OCaml website 

### ISSUE THREAD

**Issue by Goodiec** [ocaml/v2.ocaml.org#1439]
There is no alternative text on most of the images on the OCaml website. The Books page is an example, no of the images has an alt text. Adding alt text to the images will help improve accessibility for the visually impaired. In cases where the link to the image isn't correct or was changed and not updated, just the broken image icon gets displayed which doesn't communicate the content of the image that is supposed to be there to the visually impaired as seen in the image below:

![no-image-sym](https://user-images.githubusercontent.com/15726413/113709989-0f157200-96db-11eb-8a3b-4f1ef27668dc.PNG)

If this is approved, I can go ahead and add descriptive alt texts to the images.

**Comment by gs0510** [comment]
This is a good catch @Goodiec! You can go ahead and work on this :)

**Comment by ghost** [comment]
If the image is purely decorative, some guides advise using `alt=""` to indicate that the image has no useful content. This may apply to some of the images that you encounter.

**Comment by Goodiec** [comment]
> If the image is purely decorative, some guides advise using `alt=""` to indicate that the image has no useful content. This may apply to some of the images that you encounter.

I will keep that in mind and also research more on that, thank you @kanishka-work

**Comment by bastianjakobi** [comment]
@Goodiec are you still working on this?

**Comment by Goodiec** [comment]
> @Goodiec are you still working on this?

Yes, but we can do it together if you are interested in working on this. Identify a page that this issue affects and then go ahead to fix it and create a pull request referencing this issue. I started with the book page but haven't created a pull request yet so you can pick another page asides from that one first just ensure you let me know which one it is so I don't do it also. @gs0510 is it OK if we work on this together?

**Comment by gs0510** [comment]
Sure, that should be okay @Goodiec
